docs.bruno/bruno.json

```diff
@@ -0,0 +1,5 @@
+{
+  "version": "1",
+  "name": "kalvium ASAP",
+  "type": "collection"
```

Comment:
Consider adding a description field to the JSON file to provide more context about the collection. [medium]